app/views/docs/queries.phtml

```diff
@@ -0,0 +1,500 @@
+<p>Many list endpoints in Appwrite allow you to filter, sort, and paginate results using queries. Appwrite provides a common set of syntax to build queries.</p>
+
+<h2><a href="/docs/queries#query-compatibility" id="queryable-compatibility">Query Compatibility</a></h2>
+<p>Each list endpoint supports different query operations. You can find the supported query methods and attributes in the <b>References</b> section of the Appwrite documentation.</p>
+
+<h2><a href="/docs/queries#query-class" id="query-class">Query Class</a></h2>
+<p>Appwrite SDKs provide a <code>Query</code> class to help you build queries. The <code>Query</code> class has a method for each type of supported query.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 100px">Query Method</th>
+            <th>SDK Method Example</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Select</td>
+            <td><code>Query.select(["name", "title"])</code></td>
+            <td>Select which attributes should be returned from a document.</td>
```

Comment:
How does select affect the internal attributes like $id, $createdAt etc? I think these are always returned irrespective of the select.

Reply:
@stnguyen90 There's a discussion in discord. Waiting for that to resolve.

Reply:
@christyjacob4 From what I saw in that discussion, they will not be returned by default unless it's needed for offline.
https://discord.com/channels/938747207446839356/1092726106014105621

Reply:
Okay then 👍